static.go

```diff
@@ -237,6 +240,37 @@ const (
 				</form>
 			</div>
 		</section>
+		<section class="row">
+			<div id="restore" class="col-md-6">
+				<h3>Restore</h3>
+
+				<form id="user-restore" class="ro-user-restore" role="form" action="/restore" method="post">
+					<div class="feedback restore-feedback"></div>
+
+					<div class="form-group row">
+						<div class="col-md-6">
+							<label for="restore-user">User name</label>
+							<input type="text" name="Name" class="form-control" id="restore-user" placeholder="User name" required />
+						</div>
+						<div class="col-md-6">
+							<label for="restore-user-pass">Password</label>
+							<input type="password" name="Password" class="form-control" id="restore-user-pass" placeholder="Password" required />
+						</div>
+					</div>
+					<div class="form-group row">
+						<div class="col-md-6">
+							<label for="restore-user-time">Delegation Time <small>(e.g., 2h34m)</small></label>
+							<input type="text" name="Time" class="form-control" id="restore-user-time" placeholder="1h" required />
```

Comment:
The use of the placeholder attribute has caused many of the biggest frustrations from users, in no small part because it's not submitted and people think it will be. We should either put a real default value here or put something instructive like `Amount of time delegation will be valid`.